client/src/components/Sider/data/menuItems.tsx

```diff
@@ -17,8 +17,10 @@ import {
   isAnyCourseManager,
   isAnyCoursePowerUser,
   isCourseManager,
+  isDementor,
   isCourseSupervisor,
   isHirer,
+  isAnyCourseDementor,
```

Comment:
IMHO these guard names are misleading a little bit. `isDementor` sound for me exactly as "any course dementor", but it's `admin` || `courseDementor`. I'd suggest to name it somehow like `isCourseDementor` and `isDementor` to avoid this

Reply:
it seems to me not very good to use a pattern different from the name of other roles, in this case it makes sense to redo everything